javascript/javascript3/week1/index-gify.js

```diff
@@ -13,7 +13,7 @@ searchBtn.addEventListener('click', function () {
 
    const gifyUrl = "api.giphy.com/v1/gifs/search";
 
-
+// I couldn't get the api to show all GIFS under a specific keyword, What's the problem?!
    fetch("http://api.giphy.com/v1/gifs/search?q="+ gifyValue + "&api_key="+ gifyApiKey + "&limit=5")
     .then((res) => res.json())
     .then((data) => {
```

Comment:
To show all the gif's which you can see in log in line no 20, you need to make a function that takes all the gif's and shows them side by side. Right now it is showing the very first gif of the data array.

Reply:
Thanks Priyanka, I'll try to fix it :)